tests/cassandra/test_cassandra.py

```diff
@@ -0,0 +1,105 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+import threading
+import time
+from types import ListType
+import unittest
+import os
+import mock
+
+# 3p
+from nose.plugins.attrib import attr
+
+# project
+from aggregator import MetricsAggregator
+
+import logging
+
+LOG_INFO = {
+    'log_to_event_viewer': False,
+    'log_to_syslog': False,
+    'syslog_host': None,
+    'syslog_port': None,
+    'log_level': logging.INFO,
+    'disable_file_logging': True,
+    'collector_log_file': '/var/log/datadog/collector.log',
+    'forwarder_log_file': '/var/log/datadog/forwarder.log',
+    'dogstatsd_log_file': '/var/log/datadog/dogstatsd.log',
+    'jmxfetch_log_file': '/var/log/datadog/jmxfetch.log',
+    'go-metro_log_file': '/var/log/datadog/go-metro.log',
+}
+
+with mock.patch('config.get_logging_config', return_value=LOG_INFO):
+    from dogstatsd import Server
+    from jmxfetch import JMXFetch
+
+
+log = logging.getLogger('cassandra_test')
+
+STATSD_PORT = 8121
+
+
+class DummyReporter(threading.Thread):
```

Comment:
Possibly out of the scope of this PR, but it would probably be nice to refactor the `DummyReporter` to some common module to avoid this duplicated code. Being that this is an interim solution you probably don't have to worry about that right now.